src/pages/organize/[orgId]/people/duplicates/index.tsx

```diff
@@ -45,20 +59,39 @@ const DuplicatesPage: PageWithLayout = () => {
           </Typography>
         </Box>
       )}
-      {list.data && list.data.length > 0 && (
-        <Box p={1.5}>
+      {totalItems > 0 && (
+        <Box
+          ref={totalPages > 1 ? containerRef : undefined}
+          sx={{
+            p: 1.5,
+            ...(totalPages > 1 && {
+              maxHeight: '80vh',
+              overflowY: 'auto',
+            }),
+          }}
```

Comment:
I don't understand the need for this styling. I interpret the purpose of it to be to scroll only the list, fixating the pagination at the bottom of the screen, but I don't think that's necessarily a desired behavior.

On the preview server right now, because there is only one page, this styling is not in effect, and I think it works well without it: https://app-zetkin-gl9gyqnhg-zetkin.vercel.app/organize/1/people/duplicates

The pagination appears below the list, visible after scrolling down to the end of the list, and I think that's a fine behavior.

Reply:
I just realized that the pagination is _inside_ this new box, and that the way you seem to have intended it to work is to make all the content below the header, including the pagination scrollable. If that would have been the desired behavior, I think the "right" way to do that is to set the layout to `fixedHeight`. But again, I don't think it should be the desired behavior. What do you think?

Reply:
I agree with you that the styling is unnecessary and breaks the consistency with our layout. I though I need it to make the component scroll to the top when changing pages, but I got it working without it so I don't see the need for it anymore either!